stan/math/prim/mat/fun/gp_exp_quad_cov.hpp

```diff
@@ -72,8 +76,8 @@ inline
  * @tparam T_sigma type of sigma
  * @tparam T_l type of std::vector of length scale
  *
- * @param x std::vector of elements that can be used in square distance.
- *    This function assumes each element of x is the same size.
+ * @param x std::vector of Eigen column vectors of scalars.
```

Comment:
[please fix] Can you update the doc for `@tparam T_x`? Instead of "type of std::vector of elements", can you say something like: "scalar type for each of the elements"?

Reply:
[minor]
If you think to do this, great... "column vectors" should just be "vectors." We can assume the reader knows the basics of linear algebra. We tend to use "vectors" and "row vectors" (rather than "column vectors" and "row vectors").